distributed/utils_test.py

```diff
@@ -98,14 +98,14 @@
 _offload_executor.submit(lambda: None).result()  # create thread during import
 
 
-@pytest.fixture(scope="session")
+@pytest.fixture(scope="session")  # pragma: no cover
 def valid_python_script(tmpdir_factory):
     local_file = tmpdir_factory.mktemp("data").join("file.py")
     local_file.write("print('hello world!')")
     return local_file
```

Comment:
This appears to be ancient and we should remove this function. Using the [github search](https://github.com/search?p=1&q=%22valid_python_script%22&type=Code), this only pops up in distributed forks and was used in tests that have been removed two years ago in https://github.com/dask/distributed/pull/3280

Reply:
This is **ancient** stuff :)

Reply:
FYI

The github search shows a test named `test_dask_submit_cli_writes_result_to_stdout`. Finding commits in the git history which touch this test is easy with git log, e.g.

`git log -Stest_dask_submit_cli_writes_result_to_stdout` shows all commits touching that test

Reply:
that is very helpful, thank you!